.github/workflows/publish-release.yml

```diff
@@ -30,7 +30,8 @@ jobs:
       - uses: gradle/actions/setup-gradle@v4
 
       - name: Publish
-        run: ./gradlew publishAndReleaseToMavenCentral
+        # TODO: https://github.com/gradle/gradle/issues/22779
+        run: ./gradlew publishToMavenCentral --no-configuration-cache
```

Comment:
Seems like this won't be necessary after #1006 - https://github.com/vanniktech/gradle-maven-publish-plugin/issues/259#issuecomment-2994330199

Reply:
Have to remove it after bumping the project used plugin to `0.24.0`.